services/visual_recognition/v3.js

```diff
@@ -155,6 +108,12 @@ module.exports = function (RED) {
       apikey = sAPIKey || this.credentials.apikey;
       this.status({}); 
 
+      var visualRecognition = watson.visual_recognition({
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Combine this with the previous 'var' statement.](https://www.codacy.com/app/BetaWorks-NodeRED-Watson/node-red-node-watson/file/2713632089/issues/source?bid=3330694&fileBranchId=3344618#l111)